File: CSharp part1/05. Conditional-Statements/01.ExchangeIfGreater/ExchangeIfGreater.cs
﻿/* Write an if-statement that takes two double variables a and b and exchanges their values if the first
 * one is greater than the second one. As a result print the values a and b, separated by a space. */

using System;

class ExchangeIfGreater
{
    static void Main()
    {
        Console.Write("Enter a:");
        double a = double.Parse(Console.ReadLine());
        Console.Write("Enter b:");
        double b = double.Parse(Console.ReadLine());
        if(a > b)
        {
            double temp = a;
            a = b;
            b = temp;
        }
        Console.WriteLine(a + " " + b);
    }
}

File: JavaScript-Fundamentals/ConditionalStatements/01.ExchangeIfGreater/ExchangeIfGreater.js
// Write an if statement that takes two double variables a and b and exchanges their values if the first one is greater than the second.
// As a result print the values a and b, separated by a space.

var jsConsole;
var btn = document.getElementById('check');
btn.onclick = exchangeIfGreater;

function exchangeIfGreater() {
	var firstNumber = jsConsole.readFloat('#firstNumber');
	var secondNumber = jsConsole.readFloat('#secondNumber');

	if (firstNumber > secondNumber){
		var temp = firstNumber;
		firstNumber = secondNumber;
		secondNumber = temp;
	}

	jsConsole.writeLine(firstNumber + " " + secondNumber);
	console.log(firstNumber + " " + secondNumber);
}
